bevy_ios_safearea/src/plugin.rs

```diff
@@ -1,35 +1,45 @@
 use bevy::prelude::*;
-#[cfg(target_os = "ios")]
-use bevy::window::PrimaryWindow;
 
-#[derive(Copy, Clone, Debug, Default)]
+
+/// Struct providing iOS device safe area insets.
+#[derive(Resource, Clone, Debug, Default, Reflect)]
+#[reflect(Resource)]
 pub struct IosSafeArea {
+    /// The inset from the top of the screen.
+    ///
+    /// This value accounts for elements like the notch or status bar.
     pub top: f32,
+    /// The inset from the bottom of the screen.
+    ///
+    /// This value accounts for elements like the home indicator.
     pub bottom: f32,
+    /// The inset from the left side of the screen.
+    ///
+    /// This value is non-zero for devices with rounded corners or unique screen shapes.
+    pub left: f32,
+    /// The inset from the right side of the screen.
+    ///
+    /// This value is non-zero for devices with rounded corners or unique screen shapes.
+    pub right: f32,
 }
 
-#[derive(Resource, Clone, Debug, Default)]
-pub struct IosSafeAreaResource {
-    pub safe_area: Option<IosSafeArea>,
-}
-
-impl IosSafeAreaResource {
-    pub fn top(&self) -> f32 {
-        self.safe_area.map_or(0., |a| a.top)
-    }
-
-    pub fn bottom(&self) -> f32 {
-        self.safe_area.map_or(0., |a| a.bottom)
-    }
-}
-
+/// Plugin to query iOS device safe area insets.
+///
+/// # Example
+/// ```no_run
+/// use bevy::prelude::*;
+///
+/// App::new()
+///     .add_plugins((DefaultPlugins,bevy_ios_safearea::SafeAreaPlugin))
+///     .run();
+/// ```
 #[allow(dead_code)]
 #[derive(Default)]
-pub struct IosSafeAreaPlugin;
+pub struct SafeAreaPlugin;
```

Comment:
its sort of convention with the other crates that the plugins all start with `Ios`

Reply:
Restored